exercises/transpose/.meta/generator/transpose_case.rb

```diff
@@ -4,18 +4,12 @@ class TransposeCase < Generator::ExerciseCase
 
   def workload
     [
-      "input = #{indent_heredoc(input_lines, 'INPUT', 2, delimiter_mod)}",
+      "input = #{on_single_line(lines)}",
```

Comment:
`not_the_heredoc(lines)`  Because the "lines" itself indicates that it could not possibly be on a single line.  We are addressing heredoc vs string with lines.  Not that the name is necessarily good.  But the "single line" name seems to indicate we want to remove new lines, and that isn't the case.

Reply:
I've updated the name in the latest version.